client/app/dbaas/logs/detail/streams/alerts/home/alerts-home.controller.js

```diff
@@ -1,9 +1,117 @@
-class LogsStreamsAlertsHomeCtrl {
-    constructor ($stateParams) {
+class AlertsHomeCtrl {
```

Comment:
You should keep the old name, or at least have "Logs" in the name. Since we only have 1 module (managerApp), we need to avoid having name conflicts in the future.

Reply:
This is done